CHANGELOG.md

```diff
@@ -10,6 +10,11 @@
 * *BREAKING* Upgrade react v18.0.0. Refs FOLIO-3876.
 * *BREAKING* Upgrade dependent modules.
 
+## [3.0.0](https://github.com/folio-org/ui-plugin-bursar-export/tree/v3.0.0) (2023-10-02)
```

Comment:
looks like a merge error happened and 3.0.0 was duplicated. also, please add changelog for these changes